tests/EuropaComponentLibraryLoaderTest.php

```diff
@@ -42,9 +40,18 @@ public function testLoader($name, $expected)
     public function dataProvider()
     {
         return [
-            ['@namespace1/ecl-component1', 'ecl-component1.twig ecl-component2.twig ecl-component3.twig'],
-            ['@namespace2/ecl-component2', 'ecl-component2.twig ecl-component3.twig'],
-            ['@namespace2/ecl-component3', 'ecl-component3.twig'],
+            // Test without prefix.
+            ['@namespace1/component1', 'component1/component1.twig', ''],
+            // Test without prefix and with inclusions.
+            ['@namespace2/component2', 'component2/component2.twig component1/component1.twig', ''],
+            ['@namespace2/component3', 'component3/component3.twig component2/component2.twig component1/component1.twig', ''],
+            // Test without prefix and sub component.
+            ['@namespace2/component4/sub-component1', 'component4/sub-component1.twig', ''],
+            // Test with prefix.
+            ['@namespace1/component5', 'ecl-component5/component5.twig', 'ecl-'],
```

Comment:
For ECL 2.x, this case is not correct.
The test should be `['@namespace1/ecl-component5', 'ecl-component5/component5.twig', 'ecl-'],`.

e.g: https://github.com/ec-europa/ecl-twig/tree/master/src/ec/packages/ec-component-accordion
The name is `@ecl-twig/ec-component-accordion` (the package is `ecl-twig` and the prefix is `ec-component`)
the template path is `ec-component-accordion/accordion.html.twig`.
So the prefix is already present on the package name.

Reply:
We discussed on Slack about this and clarified that the ECL repo names do not mean anything for us, hence @voidtek 's approval.

Reply:
After a conversation with @ademarco, the test is correct for the future use of ecl2.